R/tab_create_modify.R

```diff
@@ -405,24 +469,14 @@ tab_row_group <- function(data,
           data = data,
           row_groups = unique(
             c(
-              process_text(group[1]),
+              id, # we are moving away from using the label
               arrange_groups_vars
             )
           )
         )
     }
   }
 
-  # Set a name for the `others` group if a
-  # name is provided
-  if (!is.null(others)) {
-    data <-
-      dt_stub_others_set(
-        data = data,
-        stub_others = process_text(others[1])
-      )
-  }
-
```

Comment:
Instead of getting rid of this, put in a deprecation warning and use `tab_options()` to actually do it.

Reply:
Done.